Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -98,10 +157,26 @@ const details = () => ({
         \\nExample:\\n 
           veryfast`,
     },
+    {
+      name: 'use10Bit',
+      type: 'boolean',
+      defaultValue: false,
+      inputUI: {
+        type: 'dropdown',
+        options: [
+          'false',
+          'true',
+        ],
+      },
+      tooltip: `Input "true" if you want to use 10 Bit colorDepth for your Encoder
+        
+        \\nExample:\\n
+        true`,
```

Comment:
"If enabled, output will be forced to 10 bit. If disabled, output will follow bit depth of original file."